app/controllers/admins_controller.rb

```diff
@@ -0,0 +1,43 @@
+class AdminsController < ApplicationController
```

Comment:
We're definitely going to want to protect this controller so that only admins can access it

Reply:
resolved

Reply:
resolved